### ISSUE TITLE
SplitButton, Error: Cannot read property 'nativeElement' of undefined

### ISSUE THREAD

**Issue by Levon770** [telerik/kendo-angular#2838]
**Describe the bug**
Using simple example from docs, with curent versions (Angular 9) there are JS error at console.

The list of installed packages from KENDO
```
"@progress/kendo-angular-buttons": "^5.4.0",
"@progress/kendo-angular-common": "^1.2.2",
"@progress/kendo-angular-dateinputs": "^4.2.2",
"@progress/kendo-angular-dialog": "^4.1.3",
"@progress/kendo-angular-dropdowns": "^4.2.6",
"@progress/kendo-angular-excel-export": "^3.1.3    ",
"@progress/kendo-angular-grid": "^4.7.0",
"@progress/kendo-angular-inputs": "^6.5.0",
"@progress/kendo-angular-intl": "^2.0.1",
"@progress/kendo-angular-l10n": "^2.0.1",
"@progress/kendo-angular-layout": "^4.2.3",
"@progress/kendo-angular-pdf-export": "^2.0.3",
"@progress/kendo-angular-popup": "^3.0.5",
"@progress/kendo-angular-treeview": "^4.1.2",
"@progress/kendo-data-query": "^1.5.3",
"@progress/kendo-drawing": "^1.6.0",
"@progress/kendo-theme-default": "^4.14.0",
```

**To Reproduce**
Look on this, to reproduce
https://stackblitz.com/edit/angular-k4mfsv

**Comment by tommut** [comment]
I am hitting the same issue using kendo-dropdownbutton/kendo-toolbar-dropdownbutton with simple example, also after upgrading to Angular 9.
<kendo-dropdownbutton [data]="data">
User Settings
</kendo-dropdownbutton>

Gives the same error:
core.js:6189 ERROR TypeError: Cannot read property 'nativeElement' of undefined
    at DropDownButtonComponent.get focused [as focused] (index.js:2688)
    at DropDownButtonComponent_HostBindings (index.js:2905)
    at setHostBindingsByExecutingExpandoInstructions (core.js:11535)
    at refreshView (core.js:11904)

**Comment by Raisolution** [comment]
Please excuse us for this. Fix for the issue is available in `v5.4.1`. Here is the [updated version of the example](https://stackblitz.com/edit/angular-v4jreb?file=src%2Fapp%2Fapp.component.html) (kendo-themes package and BrowserAnimationModule included.

**Comment by tommut** [comment]
That fix worked, thanks!  (Using kendo-dropdownburtton).
